ArmRobot/Assets/Scripts/Robot/RobotVisionCapture.cs

```diff
@@ -0,0 +1,111 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+
+public class RobotVisionCapture : MonoBehaviour
+{
+    public GameObject cube;
+    public GameObject cylinder;
+    public GameObject robot;
+   
+    public int nbMaxGameobjectScene = 10;
+    public float yAltitudeTable = 0.778f;
+    public float minimumDistanceWithoutObjects = 0.05f;
+    public float scaleObject = 0.1f;
+
+    
+    public float robotMinReach = 0.2f;
+    public float robotMaxReach = 0.5f;
```

Comment:
These `robotMinReach` and `robotMaxReach` values are also in the position randomizer classes. You should design things to avoid repeating things like this.